extensions/wikia/UserLogin/EmailConfirmationController.class.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class EmailConfirmationController extends WikiaController {
+	/**
+	 * Attempt to confirm the user's email address and show success or failure
+	 */
+	public function postEmailConfirmation() {
```

Comment:
It is, see https://github.com/Wikia/app/pull/12154/commits/88498c3e02984ba256d3ad7aef53315e042dee39